packages/runner-wrapper/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import Runner from '@dazn/chaos-squirrel-runner';
+
+export interface WrapperRunnerOptions {
+  createRunner: () => Runner;
+  wait?: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type AnyFunction = (...args: any[]) => Promise<any>;
+
+export default <Func extends AnyFunction>(
+  func: Func,
+  { createRunner, wait = true }: WrapperRunnerOptions
+): ((...args: Parameters<Func>) => ReturnType<Func>) => {
```

Comment:
Open to suggestions on improved TypeScript to maintain the types of the wrapped function - this works but is a bit messy